client/app/scripts/superdesk-search/views/item-search.html

```diff
@@ -1,41 +1,27 @@
 <form>
     <fieldset ng-if="context">
-        <label> 
-            <input type="radio" value="local" ng-model="repo.search"> 
-            In Superdesk
-        </label>
-        <div ng-show="repo.search == 'local'">
-            <div class="field">
-                <label for="ingest-collection">
-                    {{:: 'Ingest' | translate}}
-                    <input type="checkbox" id="ingest-collection" ng-model="repo.ingest" ng-checked="repo.ingest" tabindex="4">
-                </label>
-            </div>
-            <div class="field">
-                <label for="archive-collection">
-                    {{:: 'Production' | translate}}
-                    <input type="checkbox" id="archive-collection" ng-model="repo.archive" ng-checked="repo.archive" tabindex="5">
-                </label>
-            </div>
-            <div class="field">
-                <label for="published-collection">
-                    {{:: 'Published' | translate}}
-                    <input type="checkbox" id="published-collection" ng-model="repo.published" ng-checked="repo.published" tabindex="6">
-                </label>
-            </div>
-            <div class="field">
-                <label for="archived-collection">
-                    {{:: 'Archived' | translate}}
-                    <input type="checkbox" id="archived-collection" ng-model="repo.archived" ng-checked="repo.archived" tabindex="7">
-                </label>
-            </div>
+        <div class="field">
+            <label for="ingest-collection">
+                {{:: 'Ingest' | translate}}
+                <input type="checkbox" id="ingest-collection" ng-model="repo.ingest" ng-checked="repo.ingest">
+            </label>
         </div>
-    </fieldset>
-    <fieldset ng-if="context">
-        <div ng-repeat="provider in providers">
```

Comment:
Go to http://master.dev.superdesk.org/ and in global search in parametres, theres only one radio button visible, the others are not, so we concluded that it can be moved, because one radio button is useeless and the other invinsible too... Pls check and if it is needed, I will return the previous version with radio buttons.